File: samples/python-project/src/edge_cases.py
# Edge case: comment should not appear as a caller
# render_text("this is a comment, not a call")

def commented_out_call():
    pass


# Edge case: same-name function should not appear as caller of utils.render_text
def render_text(local_arg):
    return f"local: {local_arg}"


def calls_local_render_text():
    render_text("local only")

File: samples/ts-project/src/edge_cases.ts
// Edge case: comment should not appear as a caller
// capitalize("this is a comment, not a call")

function commentedOutCall() {
}

// Edge case: same-name function should not appear as caller of utils.capitalize
function capitalize(localArg: string): string {
  return "local: " + localArg;
}

function callsLocalCapitalize() {
  capitalize("local only");
}
